components/use-cases-card/use-cases-card.module.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  max-width: 558px;
+  margin: 0 36px;
+
+  @include breakpoint(small) {
+    margin: 0;
+  }
+}
+
+.logoContainer {
+  max-width: 558px;
+  min-width: 343px;
+  height: 343px;
+  background-color: $color-dark;
+  border-radius: 13px;
```

Comment:
I think these also need a border and border color. From the design:

<img width="629" alt="image" src="https://user-images.githubusercontent.com/16869061/157003900-7d887021-255e-48d1-ac91-71084c39445f.png">

Reply:
Thanks - I saw that Rashida noted on that as well.

I had updated with the border in my latest update